src/esri-loader.ts

```diff
@@ -119,6 +133,7 @@ export function loadScript(options: ILoadScriptOptions = {}): Promise<HTMLScript
         }
         // create a script object whose source points to the API
         script = createScript(options.url);
+        cssLib = createCssLib(options.css);
```

Comment:
this line and the `document.body.appendChild(cssLib);` line below should only be executed if `options.css` was passed in. So:
```js
if (!options.css) {
  document.body.appendChild(createCssLib(options.css));
}
```